ament_lint_auto/doc/index.rst

```diff
@@ -44,6 +44,26 @@ The documentation of the package `ament_cmake_test
 <https://github.com/ament/ament_cmake>`_ provides more information on testing
 in CMake ament packages.
 
+
+How to exclude linter modules with ament_lint_auto?
+------------------------------------------
+
+Linter modules can be excluded via the CMake list variable `AMENT_LINT_AUTO_EXCLUDE`.
```

Comment:
```suggestion
Linter modules can be excluded via the CMake variable `AMENT_LINT_AUTO_EXCLUDE`.
```

Reply:
Ah, I see this is how it's phrased in the docs below. Feel free to leave it as-is then.